### PR TITLE
 Update to page-metadata-parser@0.4.1 #94

### PR DESCRIPTION
**Pull request by jaredkerim**

### CODE REVIEW

**Comment by jaredkerim** [comment]
@pdehaan r?

File: app/metadata.js [hunk 1]
```diff
@@ -1,35 +1,24 @@
...
 function getDocumentMetadata(url, window) {
   const doc = window.document;
-  const metadata = getMetadata(doc);
+  const metadata = getMetadata(doc, url);
 
   const responseData = {
-    url,
-    original_url: url,
+    url: metadata.url,
+    original_url: metadata.url,
```
Comment (pdehaan) [inline app/metadata.js]: This feels wrong. Mainly, we're returning the same `metadata.url` value twice, which seems confusing, but also, the value returned from `metadata.url` may not be the original URL (if it was canonical-ized or something). Shouldn't this just be the `url` that the user specified echoed back to them?
```js
url: metadata.url,
original_url: url,
...
```

**Comment by pdehaan** [comment]
r+ w/ nit above.

**Comment by pdehaan** [comment]
**PROTIP:** Add "Fixes #94" in your original commit message, and it will auto-close the linked issue(s) once the PR gets merged.

**Pull request merged**